rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -118,10 +123,14 @@ def decode_uniswap_v2_like_swap(
                 amount=amount_out,
                 asset=crypto_asset,
             ) and
-            event.asset == A_ETH and transaction.from_address == event.location_label
+            event.asset == A_ETH and
+            transaction.from_address == event.location_label and
+            event.counterparty in (SUSHISWAP_ROUTER, UNISWAP_V2_ROUTER)
         ):
+            # this is to make sure its the amm issuing the refund and not an agreggator making a swap  # noqa: E501
```

Comment:
don't add no qa for a comment. Just put the rest to the comment below. There is space.